Comments webhooks guide

CTNicholas opened the pull request:
Guide on how to send email notifications when a comment has been created. Fixed a few typos in some other guides too.

**TODO**
- [x] It turns out we have `getComments` built in! Upgrade guide to use this.
- [x] Update to new webhooks UI
- [x] Get `@liveblocks/node` docs merged
  - [x]  https://github.com/liveblocks/liveblocks/pull/1256

marcbouchenoire commented on guides/pages/how-to-send-email-notifications-when-comments-are-created.mdx:
```diff
@@ -0,0 +1,723 @@
+
+## What we’re building
+
+In this guide we’ll be learning how to send comments notifications., and more
```
```suggestion
In this guide we’ll be learning how to send comments notifications, and more
```

marcbouchenoire commented on guides/pages/how-to-send-email-notifications-when-comments-are-created.mdx:
```diff
@@ -0,0 +1,723 @@
+
+<Banner title="Have a project ready">
+
+This guide assumes you already have a Liveblocks Comments project set up. If
```
```suggestion
This guide assumes you already have a Liveblocks Comments project set up. If you
```

marcbouchenoire commented on guides/pages/how-to-send-email-notifications-when-comments-are-created.mdx:
```diff
@@ -0,0 +1,723 @@
+
+## Create an endpoint in your project
+
+When a webhook event is triggered, it can send POST request to the back end in
```
```suggestion
When a webhook event is triggered, it can send a POST request to the back end in
```

marcbouchenoire commented on guides/pages/how-to-send-email-notifications-when-comments-are-created.mdx:
```diff
@@ -0,0 +1,723 @@
+
+## Set up webhooks on the Liveblocks dashboard
+
+To use webhooks, you need to your endpoint URL to the webhooks dashboard inside
```
Missing word, `you need to your endpoint`.

marcbouchenoire commented on guides/pages/how-to-send-email-notifications-when-comments-are-created.mdx:
```diff
@@ -0,0 +1,723 @@
+## Set up webhooks on the Liveblocks dashboard
+
+To use webhooks, you need to your endpoint URL to the webhooks dashboard inside
+our Liveblocks project, and tell the webhook to trigger when Storage document
+data has changed.
```
Needs to be referring to Comments events.

marcbouchenoire commented on guides/pages/how-to-send-email-notifications-when-comments-are-created.mdx:
```diff
@@ -0,0 +1,723 @@
+
+The [`@liveblocks/node`](/docs/api-reference/liveblocks-node) package provides
+you with a function that verifies whether the current request is a real webhook
+request from your Liveblocks. You can set this up by setting up a
```
Missing word, `from your Liveblocks`.

marcbouchenoire reviewed (changes requested):
What a great guide, amazing work @CTNicholas.

I added some comments for typos, and about some parts that still refer to syncing to a database. I also noticed that sometimes "webhook" is capitalized and sometimes not.

CTNicholas commented:
> What a great guide, amazing work @CTNicholas.
> 
> I added some comments for typos, and about some parts that still refer to syncing to a database. I also noticed that sometimes "webhook" is capitalized and sometimes not.

Thank you so much! I think I've fixed everything now.
